### ISSUE TITLE
Airplane mode iOS - When airplane mode is on, the main page says COVID Alert is offline, but there is faint white text under the message - v11

### ISSUE THREAD

**Issue by jelen-HC** [cds-snc/covid-shield-mobile#313]

**Comment by jelen-HC** [comment]
Also with airplane mode active on the main screen where it displays "COVID Alert is offline" - "COVID Alert only checks for exposure if your phone is connected to the Internet" there is white text just below this content that can barely be seen displaying "Last checked 1 minute ago" (or when ever the last time it was checked). Either needs text colour change or be removed I would imagine.

**Comment by timarney** [comment]
>  "Last checked 1 minute ago" (or when ever the last time it was checked). Either needs text colour change or be removed I would imagine.

@amazingphilippe

**Comment by srepussard** [comment]
This s happening on Android too : 
![Screenshot (Jun  30, 2020 11_36_57)](https://user-images.githubusercontent.com/47008295/86164249-4b6dcf00-bac6-11ea-8c66-f3ac5da1b99c.png)

**Comment by JuliannaR** [comment]
@timarney this font was black in previous iterations and should match https://docs.google.com/presentation/d/128gqnSP1GL-NINLCgjlNn9voeycpvtkrPWOKyMJlMWM/edit#slide=id.g8acb366c53_0_41 slide 9.
